source/_docs/configuration/templating.markdown

```diff
@@ -175,6 +176,13 @@ Other state examples:
 {% if states('sensor.train_departure_time') in ("unavailable", "unknown") %}
   {{ ... }}
 
+{% if has_value('sensor.train_departure_time') %}
+  {{ ... }}
+
+{% if sensor.train_departure_time is has_value %}
```

Comment:
what is `sensor.trains_departure_time`? as in, where does this variable come from?